From eb5b4e20425a70fdf08d264066d0afe2488781d2 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Wed, 9 Dec 2020 15:49:20 +0800
Subject: [PATCH] PCI: rockchip: dw: Fix compile error for debug function

It seem we lose struct dw_pcie  definition on the way of
rebasing the orinial commit. Fix it.

Change-Id: I0f97b20960f5647251257cf65e862aeaa39f2b7d
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 drivers/pci/controller/dwc/pcie-dw-rockchip.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pci/controller/dwc/pcie-dw-rockchip.c b/drivers/pci/controller/dwc/pcie-dw-rockchip.c
index 8f00c16f3553..ce390f415ba4 100644
--- a/drivers/pci/controller/dwc/pcie-dw-rockchip.c
+++ b/drivers/pci/controller/dwc/pcie-dw-rockchip.c
@@ -411,6 +411,7 @@ static void rk_pcie_debug_dump(struct rk_pcie *rk_pcie)
 {
 #if RK_PCIE_DBG
 	u32 loop;
+	struct dw_pcie *pci = rk_pcie->pci;
 
 	dev_info(pci->dev, "ltssm = 0x%x\n",
 		 rk_pcie_readl_apb(rk_pcie, PCIE_CLIENT_LTSSM_STATUS));
-- 
2.35.3

